drivers/led/pca9633.c

```diff
@@ -218,7 +225,8 @@ static const struct led_driver_api pca9633_led_api = {
 
 #define PCA9633_DEVICE(id)						\
 	static const struct pca9633_config pca9633_##id##_cfg = {	\
-		.i2c = I2C_DT_SPEC_INST_GET(id)				\
+		.i2c = I2C_DT_SPEC_INST_GET(id),			\
+		.disable_allcall = DT_INST_PROP(id, disable_allcall)	\
```

Comment:
One more thing.

```suggestion
		.disable_allcall = DT_INST_PROP(id, disable_allcall),	\
```

That way, there will be no need to change this line in the future if a new member will be added.